deploy/kubernetes/examples/demo-volumereplication-ear.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: replication.storage.openshift.io/v1alpha1
```

Comment:
it's problematic to put ear in the file name as its a file for clients (among others) and ear is an internal SV name.
@rivkap what name you suggest?

Reply:
demo-policy-based-volumereplication.yaml?

Reply:
Name changed to demo-policy-based-volumereplication.yaml